e2e/scaffolding/helpers.ts

```diff
@@ -461,19 +487,26 @@ export async function setEpochLength(keys: KeyringPair, epochLength: number): Pr
   }
 }
 
+export async function getNextRewardEraBlock(): Promise<number> {
+  const eraInfo = await ExtrinsicHelper.apiPromise.query.capacity.currentEraInfo();
+  const actualEraLength: number = 50;
```

Comment:
Shouldn't we be able to get this from the chain?

Reply:
@wilwade 4u

Reply:
@shannonwells Ended up being this simple, so feel free to grab this.

```suggestion
  const actualEraLength: number = ExtrinsicHelper.api.consts.capacity.eraLength.toNumber();
```

Reply:
🤦